src/Components/Graphs/Graph1.jsx

```diff
@@ -1,11 +1,21 @@
 import React, { useEffect, useState, createContext } from "react";
 import axios from "axios";
 import { Column } from "@ant-design/plots";
+// import "react-loader-spinner/dist/loader/css/react-spinner-loader.css"
+import { Oval } from 'react-loader-spinner'
 
 import "./Graph1.css";
 
 function Graph1() {
   const [data, setdata] = useState([]);
+  const [loader, setLoader] = useState(false)
+
+  useEffect(() => {
+    setLoader(true)
+    setTimeout(() => {
+      setLoader(false)
```

Comment:
What if data takes more than 5 secs to load due to slow connection?

Reply:
how to solve this sir?

Reply:
set loader to false after data fetching is done

Reply:
ok sir

Reply:
I tried it sir, but it was not working.